examples/npm_package/packages/pkg_a/BUILD.bazel

```diff
@@ -1,16 +1,29 @@
+load("@aspect_bazel_lib//lib:jq.bzl", "jq")
 load("@aspect_rules_js//npm:defs.bzl", "npm_package")
 load("@aspect_rules_js//js:defs.bzl", "js_library")
 load("@npm//:defs.bzl", "npm_link_all_packages")
 
 npm_link_all_packages(name = "node_modules")
 
+jq(
+    name = "package",
+    srcs = ["package.json"],
+    filter = "|".join([
+        # Don't directly reference $STAMP as it's only set when stamping
+        # This 'as' syntax results in $stamp being null in unstamped builds.
+        "$ARGS.named.STAMP as $stamp",
+        # Provide a default using the "alternative operator" in case $stamp is null.
+        ".version = ($stamp.STABLE_BUILD_VERSION // \"0.0.0\")",
```

Comment:
perhaps link to relevant jq documentation for users that are not familiar with the tool?